docs/cql/ddl.rst

```diff
@@ -307,6 +308,20 @@ As an alternative, you can configure your keyspace to be stored
 on Amazon S3 or another S3-compatible object store.
 See :ref:`Keyspace storage options <admin-keyspace-storage-options>` for details.
 
+.. _consistency-option:
+
+Keyspace ``consistency`` options :label-caution:`Experimental`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Experimental option that allows to configure consistency for all tables in
+a keyspace.  By default the consistency is `eventual`. But you can change
+it to strong consistency by setting the `consistency` option to `local`
```

Comment:
I think we should mention that `local` and `global` require tablets to be enabled.

Also, I'm not sure if it's fine to have this doc when `local` and `global` do nothing. We are misleading users here. Shouldn't we warn that it's work in progress and these options currently do nothing?

Reply:
After thinking about it more, I agree this doc is fine. Since an experimental feature is in the development phase, it can just as well be at the beginning of the development phase and not usable at all.